app/controllers/v1/sessions_controller.rb

```diff
@@ -89,7 +89,7 @@ def set_sentry_context_for_callback
     end
 
     def saml_settings(options = {})
-      options[:force_authn] ||= true
+      options[:force_authn] = true unless options.key?(:force_authn) && options[:force_authn] == false
```

Comment:
I'm a little confused with this line, does this say, 'set force_authn = true, unless force_authn is already defined and set to false'?

We only call `saml_settings` with either no option, or with `force_authn`, I think we should just define the function as `def saml_settings(force_authn: true)`, and then have all instances that call `saml_settings` to define `saml_settings(force_authn: false)` if they want `force_authn` to be false. Does that make sense?

Reply:
Yes, it was a fix for the `||=` that was overwriting the `force_authn=false` setting; no `force_authn=false` was getting through as-is. I updated the method per your suggestion.

Reply:
Ok, looks good to me, I tested with a quick idme and logingov user and was able to log in properly with both